extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -57,6 +64,16 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		return this._initializeDeferred.promise;
 	}
 
+	trustBook(bookTreeItem: BookTreeItem): any {
+		let hasTrustedBook = this._bookTrustManager.setBookAsTrusted(bookTreeItem);
+
+		if (hasTrustedBook) {
+			vscode.window.showInformationMessage(msgBookTrusted);
```

Comment:
All vscode/azdata API calls should go through https://github.com/microsoft/azuredatastudio/blob/master/extensions/notebook/src/common/apiWrapper.ts (I see this file isn't doing that currently, but for new stuff let's make sure we're doing that)